polyaxon/templates/cm.yaml

```diff
@@ -192,7 +192,13 @@ data:
   # Registry
   POLYAXON_REGISTRY_HOST: {{ template "docker-registry.fullname" . }}
   POLYAXON_REGISTRY_PORT: {{ (index .Values "docker-registry").service.port | quote }}
+  {{- if (index .Values "docker-registry").install }}
   POLYAXON_REGISTRY_NODE_PORT: {{ (index .Values "docker-registry").service.nodePort | quote }}
+  {{- end }}
+  {{- if (index .Values "docker-registry").registryUser }}
+  POLYAXON_REGISTRY_USER: {{ (index .Values "docker-registry").registryUser }}
+  POLYAXON_REGISTRY_PASSWORD: {{ (index .Values "docker-registry").registryPassword }}
```

Comment:
This should probably be a secret instead of env var?

Reply:
@mouradmourafiq I agree with you, I have created a secret for docker-registry when `install: false`, but the reason I used that env var is because I think `polyaxon` only looks to that env variable. I couldn't find a away to use the secret like we have for postgreSQL with `POLYAXON_K8S_DB_SECRET_NAME`. So maybe if we go through this secret feature, I should also do a PR for polyaxon/polyaxon to use a k8s secret for the docker registry when available?

Reply:
@mouradmourafiq here^

Reply:
@ricardofbarros sorry for the late answer, the secret should mimic as you said the behavior of postgreSQL. We actually do not pass the name of the registry secret to the platform, but we can change that, by adding a new env var `POLYAXON_K8S_REGISTRY_SECRET_NAME` similar to this [one](https://github.com/polyaxon/polyaxon-chart/blob/master/polyaxon/templates/cm.yaml#L38).

It should point out to the default registry secret, you will find it in the docker-registry [subchart](https://github.com/polyaxon/polyaxon-chart/blob/master/polyaxon/charts/docker-registry/templates/secrets.yaml), otherwise to the one you just created when `install is false`.

For passing `POLYAXON_REGISTRY_PASSWORD` you need to create a new entry similar to the postgres, redis, and rabbitmq, [here](https://github.com/polyaxon/polyaxon-chart/blob/master/polyaxon/templates/partials/_envs.tpl)

Reply:
@mouradmourafiq I fixed your suggestions in the last commit, I'll mark this as solved. Will continue discussion on main thread.